### DISCUSSION TITLE
Account orders in reports

### DISCUSSION THREAD

**Discussion by jimmymasaru** [firefly-iii/firefly-iii#5289] [Firefly III] [answered]
Hi,
Firstly I'd like to appreciate the great work so that I can use a self-hosted financing tool!
I've been playing around with the Reports function and noticed the accounts are not sorted in the order I set in Asset Accounts.
The report I run is at `/reports/audit/allAssetAccounts/currentMonthStart/currentMonthEnd`. I'm not too sure if I missed a setting or it's an existing issue in the code.

**Comment by JC5** [comment, marked as answer]
Nice find, I'll pick it up in the new layout.
